numba/core/typing/typeof.py

```diff
@@ -11,6 +11,12 @@
 # terminal color markup
 _termcolor = errors.termcolor()
 
+# https://github.com/numba/numba/pull/7900#issuecomment-1065026619
```

Comment:
I think it may make sense (and is safer in the long run) to include a verbatim statement of why this change is needed. We never know when Github will become obsolete and this comment will end up pointing no-where.